pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -694,3 +694,60 @@ func createNondetElementsOverXHinter(resolver hintReferenceResolver, x uint64) (
 
 	return newNondetElementsOverXHint(elementsEnd, elements, x), nil
 }
+
+// NormalizeAddress hint checks if given addr value is less than ADDR_BOUND or not
+// and writes the result as 1 or 0 to is_small
+//
+// `newNormalizeAddressHint` takes 2 arguments
+//   - `isSmall` represents the address where the result of the comparison is stored
+//   - `addr` represents the address whose value is checked against ADDR_BOUND
+func newNormalizeAddressHint(isSmall, addr hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "NormalizeAddress",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> # Verify the assumptions on the relationship between 2**250, ADDR_BOUND and PRIME.
+			//> ADDR_BOUND = ids.ADDR_BOUND % PRIME
```

Comment:
what would be the sense of this modulo here? making sure this hint is not utilised in a context with a prime too big (or too small)? but i think `PRIME` is also a constant no? anyway... goddamn python vm